app/views/controllers/account/welcome.html.erb

```diff
@@ -4,7 +4,9 @@ add_page_title(account_welcome_title(@user))
 
 <% if @user %>
   <%= :welcome_note.tp %>
-  <%= link_to(:welcome_logout_link.t, account_logout_path) %>
+  <%= button_to(:app_logout.t,
+                account_logout_path,
+                { class: "btn", id: "nav_user_logout_link" }) %>
```

Comment:
iirc with BS everywhere you use `btn` you have to also use a coloration, like `btn-default`.